scripts/binaryninja/manticore_viz/plugin.json

```diff
@@ -0,0 +1,15 @@
+{
+	"plugin": {
+		"name": "ManticoreTrace",
+		"type": ["ui"],
+		"api": "python2",
+		"description": "Manticore Trace Visualization.",
+		"longdescription": "This plugin visualizes Manticore execution traces.\n",
+		"license": {
```

Comment:
Perhaps remove the license overall and keep the json as it is recommended by the binja plugin format?